dump.py

```diff
@@ -15,17 +16,28 @@ def fatal(reason):
     sys.exit(-1)
 
 
-def find_app(app_name_or_id, device_id):
+def find_app(app_name_or_id, device_id, device_ip):
     if device_id is None:
-        dev = frida.get_usb_device()
+        if device_ip is None:
+            dev = frida.get_usb_device()
+        else:
+            dManager = frida.get_device_manager()
```

Comment:
I don't see `dManager` makes any sense here

Reply:
thought it would fix some connection problems. but retestet and now works fine without. so just removed.